service/user.service.js

```diff
@@ -4,14 +4,22 @@ const tokenService = require("../service/token.service");
 const UserDTO = require("../dto/user.dto");
 const ApiError = require("../helper/api.error");
 
-const signUp = async (name, email, password) => {
+const signUp = async (first_name, last_name, phone_number, email, password) => {
     const candidate = await UserModel.findOne({ email });
```

Comment:
@ikadushkina You request a whole user object to check either a user with this email exists or not. It can be little bit optimised, if you will request only count of docs `UserModel.findOne({ email })` or use projection here to get only _id field `UserModel.findOne({ email }, { _id: 1 })`. This optimisation will lighten the load on the server little bit.